runtime/mount/agent.py

```diff
@@ -3254,6 +3262,11 @@ async def accept(self) -> None:
         elif msg_type == "agent_action_response":
             print(f"[agent] {msg.get('action', 'unknown')} -> {msg.get('status', 'unknown')}")
             await self.handle_action_response(msg)
+        elif msg_type == "agent_behavior_update":
+            proactive_behavior_enabled = msg.get("proactive_behavior_enabled")
+            if proactive_behavior_enabled is not None:
+                self.proactive_behavior_enabled = bool(proactive_behavior_enabled)
+                print(f"[agent] Proactive mode set to {self.proactive_behavior_enabled}")
```

Comment:
Cleaner to have it submitted on every user request and to store the latest preference for current model behavior?